docs/en/observability/synthetics-command-reference.asciidoc

```diff
@@ -101,21 +123,65 @@ However there are some limitations when using external packages:
 * Native node modules will not work as expected due to platform inconsistency. 
 ====
 
-`--auth`::
+`--auth <string>`::
 API key used for {kibana-ref}/api-keys.html[{kib} authentication].
 +
 If you are pushing to a <<private-locations,{private-location}>>, you must use an API key generated in 8.4 or higher.
 +
 To create an API key, you must be logged into {kib} as a user with the privileges described in
 <<synthetics-role-write>>.
 
-`--url`::
-The {kib} URL for the deployment to which you want to upload the monitors.
-
-`--project`::
+`--project <string>`::
 A unique id associated with your project.
 It will be used for logically grouping monitors.
++
 If you used <<elastic-synthetics-init-command, `init` to create a project>>, this is the `<name-of-project>` you specified.
++
+This can also be set in the configuration file using
+<<synthetics-configuration-project,`project.id`>>.
+The value defined via the CLI will take precedence.
+
+`--url <string>`::
+The {kib} URL for the deployment to which you want to upload the monitors.
++
+This can also be set in the configuration file using
+<<synthetics-configuration-project,`project.url`>>.
+The value defined via the CLI will take precedence.
+
+`--space <string>`::
+The identifier of the target {kibana-ref}/xpack-spaces.html[{kib} space] for the pushed monitors.
+Spaces help you organize pushed monitors.
+Defaults to `default`.
++
+This can also be set in the configuration file using
+<<synthetics-configuration-project,`project.space`>>.
+The value defined via the CLI will take precedence.
+
+`--schedule <number>`::
+The interval (in minutes) at which the monitor should run.
++
+This can also be set in the configuration file using
+<<synthetics-configuration-monitor,`monitor.schedule`>>.
+The value defined via the CLI will take precedence.
+
+https://github.com/elastic/synthetics/blob/{synthetics_version}/src/locations/public-locations.ts#L28-L37[`--locations Array<SyntheticsLocationsType>`]::
+Where to deploy the monitor. Monitors can be deployed in multiple locations so that you can detect differences in availability and response times across those locations.
++
+To list available locations, refer to <<elastic-synthetics-locations-command>>.
++
+Locations passed will be merged with locations defined in your
+<<synthetics-configuration-monitor,`synthetics.config.js` file>> and
+<<synthetics-monitor-use,when using `monitor.use`>> in individual journey files.
```

Comment:
Is this true?

Reply:
I just tested this out and the CLI setting for `--locations`:
* overwrites the location set globally via `monitor.locations` in `synthetics.config.ts`
* does not override the location set directly against the individual monitor via `monitor.use.locations`
* does not get _merged_ with either (it will either override or do nothing)

Reply:
Yes this is working as expected. Priority order
1. Individual monitor config `monitor.use(locations)`
2. If set via CLI --locations
3. Global config from `synthetics.config.ts`

Nothing gets merged, everything will be overridden.